activation/malfeasance2.go

```diff
@@ -50,3 +50,8 @@ func (p *MalfeasanceHandlerV2) Publish(ctx context.Context, id types.NodeID, pro
 	// TODO(mafa): implement me
 	return nil
 }
+
+func (p *MalfeasanceHandlerV2) Republish(ctx context.Context, id types.NodeID) error {
+	// TODO(mafa): implement me
```

Comment:
IMO you should add a gh issue for that to keep track of it.

Reply:
The issue is this: https://github.com/spacemeshos/go-spacemesh/issues/6143 and I'm already implementing these methods in https://github.com/spacemeshos/go-spacemesh/pull/6307 😊